assets/src/blocks/HubspotForm/HubspotFormEditor.js

```diff
@@ -0,0 +1,161 @@
+import { Fragment, useEffect, useRef } from '@wordpress/element';
+import { MediaUpload, MediaUploadCheck, RichText } from '@wordpress/block-editor';
+import { Button } from '@wordpress/components';
+import { Sidebar } from './HubspotFormSidebar';
+import { useToAttribute } from './hooks/useToAttribute';
+import { useBackgroundImage } from './hooks/useBackgroundImage';
+import { getStyleFromClassName } from '../getStyleFromClassName';
+
+const { __ } = wp.i18n;
+
+export const HubspotFormEditor = ({
+  attributes: {
+    block_background_image_id,
```

Comment:
If this is the main background for the block, we could just use `backgroundImage`.

These attributes can be camelCase, btw. I think the snake_case format in our code is a leftover from the shortcodes. WordPress itself nowadays uses camelCase for all block attributes.